src/core/packages/http/server-internal/src/http_config.ts

```diff
@@ -138,7 +138,7 @@ const configSchema = schema.object(
         defaultValue: 'http1',
       })
     ),
-    prototypeHardening: schema.boolean({ defaultValue: false }),
+    prototypeHardening: schema.boolean({ defaultValue: true }),
```

Comment:
Now that `prototypeHardening` is enabled by default, do we still need the overrides in FTR configs?

https://github.com/elastic/kibana/blob/00ad46beb649bfd5d14315f849b2733022e256dc/x-pack/platform/test/serverless/shared/config.base.ts#L110

https://github.com/elastic/kibana/blob/00ad46beb649bfd5d14315f849b2733022e256dc/src/platform/packages/shared/kbn-scout/src/config/serverless/serverless.base.config.ts#L102

Reply:
Great catch, we do not. I'll update to remove the overrides